nca/NetworkConfig/ResourcesHandler.py

```diff
@@ -69,24 +69,24 @@ def get_full_livesim_resource_path(livesim_resource_path):
         return os.path.join(current_path, livesim_resource_path)
 
     @staticmethod
-    def get_relevant_livesim_resources_paths_by_labels_matching(livesim_resource_path, missing_resource_labels_dict):
+    def get_relevant_livesim_resources_paths_by_labels_matching(livesim_resource_path, missing_resource_labels):
         """
         check by labels matching if one of the livesim resources has matching labels for a resource referenced by one
         of the parsed policies. If yes, return its path to be added to the configuration, to enable the analysis.
         :param str livesim_resource_path: a path to the relevant livesim dir to check for resources
-        :param dict missing_resource_labels_dict: the labels from parsed policy in the config for
+        :param list missing_resource_labels: the labels from parsed policy in the config for
```

Comment:
isn't `missing_resource_labels` a set of pairs rather than a list?

Reply:
Right. Updated.